projects/rero/ng-core/src/lib/record/search/record-search.component.html

```diff
@@ -160,6 +160,12 @@ <h5 *ngIf="typesInTabs.length == 1 && showLabel">
         </div>
       </div>
       <div class="row">
+        <div class="col-sm-12">
+          <ng-core-list-filters
+          [aggregations]="aggregations"
+          [aggregationsFilters]="aggregationsFilters">
+          </ng-core-list-filters>
+        </div>
```

Comment:
is the div is really needed ? you should set classes on `ng-core-list-filters` element.
Additionally, use `col-12` instead of `col-sm-12` ; `-sm` classes is useful in combination with `-md` or `-lg` classes ; otherwise use the default `col-x` corresponding class